source/includes/fact-system-env-vars.rst

```diff
@@ -0,0 +1,81 @@
+.. list-table::
+   :widths: 30 10 40 10 10
+   :stub-columns: 1
+   :header-rows: 1
+
+   * - Environment Variable
+     - Data Type
+     - Description
+     - Default Value (Atlas-hosted)
+     - Default Value (On Premises)
+   * - ``log_level``
+     - integer
+     - Specifies the logging level for |bi-short|:
+
+       * ``-1``: Do not log
+       * ``0``: Log only messages for database users, such as basic
+         ``mongosqld`` events and state changes.
+       * ``1``: Log only messages for ``mongosqld`` administrators.
+       * ``2``: For internal use only.
+     - ``0``
```

Comment:
If I'm reading this right, it looks like the values here are reversed - Atlas-hosted BIC has a default log level of 2.

Reply:
Yup. Fixed.